### DISCUSSION TITLE
README mentions a ci-checked example but where is the CI result?

### DISCUSSION THREAD

**Discussion by rcambrj** [nix-community/raspberry-pi-nix#87] [Q&A] [answered]
The README mentions a ci-checked example:

> See the `rpi-example` config in this flake for a CI-checked example.

https://github.com/nix-community/raspberry-pi-nix?tab=readme-ov-file#example

Would you kindly elaborate on what, where and how?

**Comment by tstat** [comment, marked as answer]
That's ambiguously phrased. It's not CI-checked in the sense that CI boots the image. It's CI-checked in that it is an image produced using this repo that CI compiles. 

The motivation for creating it was that I got a number of bug reports or questions around how to use this repo to produce an image, and so I created a CI-checked reference example.

It is defined [here](https://github.com/nix-community/raspberry-pi-nix/blob/beb01f006a795fa1c3d46ca2a7a3dd199d919b0b/flake.nix#L64-L69), exposed as an output [here](https://github.com/nix-community/raspberry-pi-nix/blob/beb01f006a795fa1c3d46ca2a7a3dd199d919b0b/flake.nix#L84), and added as a check [here](https://github.com/nix-community/raspberry-pi-nix/blob/beb01f006a795fa1c3d46ca2a7a3dd199d919b0b/flake.nix#L70). The results show up in CI, for example [here](https://buildbot.nix-community.org/#/builders/2641/builds/41).

**Reply by rcambrj** [reply]
Aw that's great, thanks for taking the time to explain this.
I didn't know about [buildbot](https://buildbot.nix-community.org/#/projects/15).
It may be worth throwing a link into the README.
Although I suppose this discussion will now serve as a link for inquiring minds :)
